app/graphql/types/system.rb

```diff
@@ -54,6 +54,18 @@ def rules_failed(args = {})
       end
     end
 
+    def test_result_profiles
+      ::CollectionLoader.for(object.class, :test_result_profiles)
+                        .load(object)
+    end
+
+    def policies
+      ::CollectionLoader.for(object.class, :policies)
+                        .load(object).then do |policies|
+        policies.map(&:initial_profile)
+      end
+    end
+
```

Comment:
You're overriding what's in `app/graphql/types/concerns/system_profiles.rb`, so we should just change those methods instead of adding them here.